gst/tensor_mux/gsttensormux.c

```diff
@@ -268,11 +267,11 @@ gst_tensor_mux_request_new_pad (GstElement * element, GstPadTemplate * templ,
         sizeof (GstTensorCollectPadData), NULL, TRUE);
     tensormuxpad->pad = newpad;
     gst_pad_set_element_private (newpad, tensormuxpad);
-    tensor_mux->tensors_config.info.num_tensors++;
```

Comment:
Where do you increment ```tensor_mux->tensors_config.info.num_tensors```, then?
(Don't you need to do ```... += blahblah``` instead of ```...++;``` ?)

Reply:
@myungjoo Please refer https://github.com/nnsuite/nnstreamer/pull/991#discussion_r242038181.